app/helpers/application_helper.rb

```diff
@@ -120,4 +120,15 @@ def insert_extras(body)
     body
   end
 
+  def render_map(lat, lon, items)
+    
+      var map = L.map('map').setView([lat,lon], 10);
+      L.tileLayer("https://a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map)
+
+      @items.each do |item|
```

Comment:
See, here we're back to Ruby code. I guess we could alternatively put all the Ruby and JS into just a template like `/app/views/maps/_leaflet.html.erb` (instead of `.js.erb`) and include the map HTML itself along with the javascript in that file, and be able to insert values using ERB:

```ruby
<%= render partial: "maps/leaflet", locals: { lat: ..., lon: ..., items: @maps } %>
```

Then we'd have access to those locally specified variables in the template (Rails calls these "partials" since they aren't a whole page):

```ruby
var lat = <%= @lat %>,
      lon = <%= @lon %>;

<% @items.each do |item| %>
   ...
<% end %>
```
Does that make sense? Sorry if I led us down the wrong path with the helper idea... but later we could simplify this using a helper, maybe.